example/android/app/build.gradle

```diff
@@ -1,5 +1,5 @@
 apply plugin: "com.android.application"
-apply plugin: 'com.google.gms.google-services'
+// apply plugin: 'com.google.gms.google-services'
```

Comment:
I added the dependencies back but we can't use this plugin without a `google-services.json` 
<img width="800" alt="image" src="https://user-images.githubusercontent.com/50601149/174346162-84954a8e-b6c5-4732-a3ed-02e0f6eb4348.png">

Reply:
I'm curious why is it doing this task `app:processDebugGoogleServices` since there's no Gradle plugin anymore. Do we need the google gradle plugin in the `android-kotlin` lib?